scripts/migration/initialize_menas.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from pymongo.database import Database
+from typing import List
+
+from menas_db import MenasDb, MenasDbVersionError
+from constants import *
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        prog='initialize_menas',
+        description='Menas MongoDB initialization script.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter  # prints default values, too, on help (-h)
+    )
+
+    parser.add_argument('-n', '--dryrun', action='store_true', default=DEFAULT_DRYRUN,
+                        help="if specified, skip the actual synchronization, just print what would be copied over.")
+    parser.add_argument('-v', '--verbose', action="store_true", default=DEFAULT_VERBOSE,
+                        help="prints extra information while running.")
+
+    parser.add_argument('target', metavar="TARGET", help="connection string for target MongoDB")
+
+    parser.add_argument('-t', '--target-database', dest="targetdb", default=DEFAULT_DB_NAME,
+                        help="Name of db on target to migrate to.")
+
+    return parser.parse_args()
+
+
+def initialize_menas_db(menas_db: MenasDb) -> None:
+
+    # create db_version record
+    print("'db_version' initialization:")
+    create_db_version(menas_db.mongodb)
```

Comment:
Please move this to be the last creation of the init script.

Reply:
Done.